site/en/blog/permissions-chip/index.md

```diff
@@ -29,36 +30,31 @@ up to version 96. (You can try this and other permissions on our demo site
 Chrome's telemetry data proves that a lot of permission prompts are ignored. You can explore the
 [notification permission data](https://developers.google.com/web/updates/2020/02/notification-permission-data-in-crux)
 in the Chrome UX report yourself. For now, consider the table below that shows
-how Windows users reacted to the geolocation and the notification prompts on sites in an accumulated way:
+how Windows users reacted to the notification prompt on sites in an accumulated way:
```

Comment:
If removing the geolocation metrics, we should still be able to say we see similar dismiss / ignore rates for geolocation - being the second most requested permission type.

Reply:
Added.